incubator/busbar/values.yaml

```diff
@@ -0,0 +1,58 @@
+## Busbar default Values
+##
+
+## Images
+image:
+  busbar:
+    repository: busbario
+    name: busbar
```

Comment:
Instead of separating `repository` and `name` it should be called `repository` and have a value of `busbario/busbar`. The includes in other files should be updated as well.

Reply:
Cool... Fixing it shortly! =)